src/ui/components/results/paginationcomponent.js

```diff
@@ -249,6 +249,12 @@ export default class PaginationComponent extends Component {
       const num = { number: i };
       if (i === pageNumber) {
         num.active = true;
+        if (this._maxVisiblePagesMobile === 1) {
+          num.onePageMobile = '-onepage-mobile';
```

Comment:
Could we turn this into a boolean (like below)? I am of the camp that there is more value in knowing whether this is true or false, and then in the template, we can add classes and logic as needed.